### PR TITLE
Added some missing bots!!

### PR DESCRIPTION
**Pull request by devbish**
I've added twitter bot, Whatsapp bot Linkedin bot, quora bot, Baidu bot, Apple bot, Telegram bot, slack bot, and Snapchat bot...

### CODE REVIEW

File: list.json [hunk 1]
```diff
@@ -870,5 +870,14 @@
   "zgrab",
   "Zippy",
   "ZnajdzFoto/Image",
-  "ZyBorg"
+  "ZyBorg",
+  "WhatsApp",
```
Comment (omrilotan) [inline list.json:874]: Covered by entry "whatsapp"
https://github.com/gorangajic/isbot/blob/master/list.json#L805

File: list.json [hunk 2]
```diff
@@ -870,5 +870,14 @@
...
+  "Twitterbot/1.0",
```
Comment (omrilotan) [inline list.json:875]: Covered by entry "bot"
https://github.com/gorangajic/isbot/blob/master/list.json#L122

File: list.json [hunk 3]
```diff
@@ -870,5 +870,14 @@
...
+  "LinkedInBot",
```
Comment (omrilotan) [inline list.json:876]: Covered by entry "bot"
https://github.com/gorangajic/isbot/blob/master/list.json#L122

File: list.json [hunk 4]
```diff
@@ -870,5 +870,14 @@
...
+  "Quora",
+  "Baidu",
+  "Applebot 0.1",
```
Comment (omrilotan) [inline list.json:879]: Covered by entry "bot"
https://github.com/gorangajic/isbot/blob/master/list.json#L122

File: list.json [hunk 5]
```diff
@@ -870,5 +870,14 @@
...
+  "TelegramBot",
```
Comment (omrilotan) [inline list.json:880]: Covered by entry "bot"
https://github.com/gorangajic/isbot/blob/master/list.json#L122

File: list.json [hunk 6]
```diff
@@ -870,5 +870,14 @@
...
+  "Slack",
```
Comment (omrilotan) [inline list.json:881]: Could you specify the user agent string in the tests? Slack link checker (`Slackbot-LinkExpanding 1.0 (+https://api.slack.com/robots)`) is already covered by entry "bot"
https://github.com/gorangajic/isbot/blob/master/list.json#L122

File: list.json [hunk 7]
```diff
@@ -870,5 +870,14 @@
...
+  "Snapchat",
```
Comment (omrilotan) [inline list.json:882]: Could you specify which user agent string you wich to check here? I don't think the in-app browser (`Mozilla/5.0 (iPhone; CPU iPhone OS 13_3_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.5 Mobile/15E148 Snapchat/10.77.5.59 (like Safari/604.1)`) should be considered a bot

Comment (devbish) [reply list.json:882]: You know the social media apps crawl the meta-tags(Open Graph) on a website when a user posted any links, therefore, I've added these links.

**Comment by omrilotan** [comment]
@devbish Sorry for the late response. I've been looking into the lists in user-agents.net/bots and others. I've started working on incorporating them into testing so we can get updates effortlessly.

I think it's a great chance to address issue #65 and rethink the existing lists. This will be an exciting version and using dynamic lists will allow it to be optimised as well.

Thank you for your patience. I've been working on this for a few evenings this week but progress is a bit slow these times ;) I aim to release a pull request next week and would love for your review, I'll ping you so you can review if you're available. I think it can have great benefits!

**Comment by omrilotan** [comment]
Addressed in 2dc2494 [(version 3)](https://github.com/gorangajic/isbot/releases/tag/v3.0.0).
Thanks!

**Pull request closed without merging**